src/behaviours/update-app/installing-update-using-tray.test.ts

```diff
@@ -300,7 +300,12 @@ describe("installing update using tray", () => {
           });
 
           it("asks user to install update immediately", () => {
-            expect(askBooleanMock).toHaveBeenCalledWith("Do you want to install update some-version?");
+            expect(askBooleanMock).toHaveBeenCalledWith({
+              id: "install-update",
+              title: "Update Available",
+              question:
+                "Version some-version of Lens IDE is available and ready to be installed. Would you like to update now?",
+            });
```

Comment:
I don't think sort of behaviour should exist. It ties the implementations too close together. Main shouldn't be making UI/UX decisions on the renderer's behalf.

Reply:
Let's keep the discussion for this topic here: https://github.com/lensapp/lens/pull/5433/files/18d13a23ac30c5617facd56aff646383a4676931#r880880811